addons/hr/wizard/mail_activity_schedule.py

```diff
@@ -28,6 +30,22 @@ def _compute_plan_department_filterable(self):
         for wizard in self:
             wizard.plan_department_filterable = wizard.res_model == 'hr.employee'
 
+    @api.depends('plan_date', 'plan_id')
+    def _compute_plan_summary(self):
+        if not self.env.context.get('sort_by_responsible', False):
+            return super()._compute_plan_summary()
+        self.plan_summary = False
+        responsible_value_to_label = dict(self.env['mail.activity.plan.template']._fields['responsible_type'].selection)
+        for scheduler in self:
+            templates_by_responsible_type = defaultdict(list)
+            for template in scheduler.plan_id.template_ids:
+                responsible = responsible_value_to_label[template.responsible_type]
+                templates_by_responsible_type[responsible].append(template)
```

Comment:
`scheduler.plan_id.template.grouped('responsible_type')`? And you could even probably do it ealier, on self instead of `scheduler`

Reply:
You just have one responsible per template so I don't understand the grouped. Do you have a example with grouped? :D

Reply:
ouah :o I was not aware about that :o thanks :raised_hands: